internal/server/v0/triple/golden/get_triples/place_type.json

```diff
@@ -92,7 +92,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/sgvysq3"
```

Comment:
Is this provenance diff expected? I believe it's from this import:
https://github.com/datacommonsorg/data/pull/736/files#diff-5ebef208962874063879510d265089ec2da98f628b22151afbf29ef23df7a4c3

which defines country nodes to use the resolver.

/cc @pulkit-s

Reply:
The provenance is non-deterministic. I think we need a proper fix for this longer term.